docs/guides/feature-flags.rst

```diff
@@ -12,16 +12,12 @@ or disable one or more of these featured flags for a particular project.
 Available Flags
 ---------------
 
-``USE_SPHINX_LATEST``: :featureflags:`USE_SPHINX_LATEST`
-
-``ALLOW_DEPRECATED_WEBHOOKS``: :featureflags:`ALLOW_DEPRECATED_WEBHOOKS`
-
 ``PIP_ALWAYS_UPGRADE``: :featureflags:`PIP_ALWAYS_UPGRADE`
 
-``SKIP_SUBMODULES``: :featureflags:`SKIP_SUBMODULES`
-
 ``DONT_OVERWRITE_SPHINX_CONTEXT``: :featureflags:`DONT_OVERWRITE_SPHINX_CONTEXT`
 
-``MKDOCS_THEME_RTD``: :featureflags:`MKDOCS_THEME_RTD`
-
 ``DONT_SHALLOW_CLONE``: :featureflags:`DONT_SHALLOW_CLONE`
+
+``USE_TESTING_BUILD_IMAGE``: :featureflags:`USE_TESTING_BUILD_IMAGE`
+
+``SHARE_SPHINX_DOCTREE``: :featureflags:`SHARE_SPHINX_DOCTREE`
```

Comment:
I think this one was only for testing, and we haven't seen any failure related to this. Should we sample more projects or just remove the flag and activate the feature for all projects?

Reply:
I'm not really sure. We could sample more projects before making it the default. Although, I think that decision is not related with this PR and in the meantime we should expose it to users to let them know that it exists.

It's a useful one when generating the doctree takes some time, so being able to share it will reduce their build time.

Reply:
> we should expose it to users to let them know that it exists.

I think that decision is part of the core team, not to the user. Users shouldn't know anything about that.

Reply:
Why?

Reply:
This isn't a setting that will change the build result in any way, or at least it shouldn't. It will safe resources, maybe, but that's not something the user will care about so much, that's more our concern. It's like having a cache for pip, that's useful for us, not something the user cares about so much.

Reply:
Good point. I will remove it from this docs then.